src/sentry/migrations/0915_backfill_open_periods.py

```diff
@@ -62,70 +57,46 @@ def get_open_periods_for_group(
     activities: list[Any],
     GroupOpenPeriod: Any,
 ) -> list[Any]:
-    # Filter to REGRESSION and SET_RESOLVED_XX activties to find the bounds of each open period.
-    # The only UNRESOLVED activity we would care about is the first UNRESOLVED activity for the group creation,
-    # but we don't create an entry for that.
+    # No activities means the group has been open since the first_seen date
+    if not activities:
+        return [
+            GroupOpenPeriod(
+                group_id=group_id,
+                project_id=project_id,
+                date_started=first_seen,
+            )
+        ]
```

Comment:
This part is probably unnecessary, since if there are no activities, we'll skip the for loop and produce this same `GroupOpenPeriod` in the if statement after

Reply:
good point, gonna leave it here so it's explicit just to make it easier to debug/read for myself